<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogCadastre_se</class>
 <widget class="QDialog" name="DialogCadastre_se">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>297</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="lblNome">
       <property name="text">
        <string>Nome:</string>
       </property>
       <property name="buddy">
        <cstring>lineNome</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineNome">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblSobrenome">
       <property name="text">
        <string>Sobrenome:</string>
       </property>
       <property name="buddy">
        <cstring>lineSobrenome</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineSobrenome">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lblEmail">
       <property name="text">
        <string>Email:</string>
       </property>
       <property name="buddy">
        <cstring>lineEmail</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEmail">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lblSenha">
       <property name="text">
        <string>Senha:</string>
       </property>
       <property name="buddy">
        <cstring>lineSenha</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineSenha">
       <property name="inputMethodHints">
        <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="lblRepitaSenha">
       <property name="text">
        <string>Repita a senha:</string>
       </property>
       <property name="buddy">
        <cstring>lineRepitaSenha</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineRepitaSenha">
       <property name="inputMethodHints">
        <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="btnEscolhaImagem">
         <property name="text">
          <string>Escolha uma imagem</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblFoto">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>150</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>150</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>150</width>
           <height>150</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../../../TCC_FTT/fontes/QTCC_Qt/QTCC_Qt.qrc">:/images/1415099481_user-64.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="buddy">
          <cstring>btnEscolhaImagem</cstring>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="lblTextoPerfil">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Texto de perfil (opcional):</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <property name="buddy">
        <cstring>lineTextoStatus</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineTextoStatus">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnCancelar">
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCadastrar">
       <property name="text">
        <string>Cadastrar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../TCC_FTT/fontes/QTCC_Qt/QTCC_Qt.qrc"/>
 </resources>
 <connections/>
</ui>
